backend/infrahub/core/schema/definitions/internal.py

```diff
@@ -525,6 +525,20 @@ def to_dict(self) -> dict[str, Any]:
             optional=True,
             extra={"update": UpdateSupport.VALIDATE_CONSTRAINT},
         ),
+        SchemaAttribute(
```

Comment:
I think if we have `min_value` and `max_value` under `parameters`, there is no need to have them at the root of attribute as well.

Reply:
I agree, I'd say the purpose of the `parameters` is to remove these extra attributes from the main schema. Today according to the schema we have `min_length` on all attributes and that doesn't make any sense for attributes of kind=Boolean etc.